lib/definitions/cloud-services/accounts-cloud-service.d.ts

```diff
@@ -0,0 +1,54 @@
+interface IAccountsCloudService {
+	getAccounts(): Promise<IAccount[]>
+	getUsageInfo(accountId: string): Promise<IUsageInfo[]>;
```

Comment:
This could be extracted into a separate interface (for example `IGetUsageInfo` or `IAccountsServiceBase`) and be `extend`-ed here and in `IAccountsService`